bull-bean-transformer/src/test/java/com/hotels/beans/transformer/ImmutableObjectTransformationTest.java

```diff
@@ -500,6 +513,33 @@ public void testTransformRaiseAnExceptionIfTheTransformerFunctionIsNotValid() {
 
     }
 
+    /**
+     * Test that an {@link InvalidBeanException} is raised if the class instantiation fails when the method {@code injectValues()} is called.
+     * @throws Exception if something goes wrong
+     */
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testInjectValuesThrowsException() throws Exception {
+        //GIVEN
+        InvalidBeanException expectedException = new InvalidBeanException("Dummy exception");
+        TransformerImpl underTestMock = spy(TransformerImpl.class);
+        ClassUtils classUtils = mock(ClassUtils.class);
+        Constructor<MutableToFooSimple> constructor = classUtils.getAllArgsConstructor(MutableToFooSimple.class);
+        when(classUtils.getInstance(constructor)).thenThrow(InvalidBeanException.class);
+        when(classUtils.getConstructorParameters(constructor)).thenReturn(new Parameter[] {});
+        when(classUtils.areParameterNamesAvailable(constructor)).thenReturn(true);
+        doReturn(expectedException).when(underTestMock).handleInjectionException(any(), any(), any(), any(), any(), anyBoolean(), any());
+        setField(underTestMock, CLASS_UTILS_FIELD_NAME, classUtils);
+        Method injectValuesMethod =
+            TransformerImpl.class.getDeclaredMethod(INJECT_VALUES_METHOD_NAME, Object.class, Class.class, Constructor.class, String.class, boolean.class);
+        injectValuesMethod.setAccessible(true);
+        //WHEN
```

Comment:
please add an empty line before "//WHEN" and "//THEN"

Reply:
This is done.